back-end-side/Controllers/ReportReader.cs

```diff
@@ -82,9 +104,9 @@ public ReportReader(int bank, IFormFile fileData)
                 {
                     records.ElementAt(i).Amount /= 100;
 
-                    if (records.ElementAt(i).PaymentType.Equals("K"))
+                    if (records.ElementAt(i).PaymentType.Equals("C"))
```

Comment:
Why you changed it to "C"?

Reply:
Because SEB integrates whether its credit or debit with C\D instead of K\D like Swedbank and Paysera